crates/driver/src/tests/cases/mod.rs

```diff
@@ -18,30 +21,55 @@ pub mod solver_balance;
 /// Example solver name.
 const SOLVER_NAME: &str = "test1";
 
+// All values below in WEI
+
 /// The default surplus factor. Set to a high value to ensure a positive score
 /// by default. Use a surplus factor of 1 if you want to test negative scores.
-pub const DEFAULT_SURPLUS_FACTOR: u64 = 10000000000u64;
+pub const DEFAULT_SURPLUS_FACTOR: f64 = 1e-8;
 
-pub const DEFAULT_POOL_AMOUNT_A: u128 = 100000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_B: u128 = 6000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_C: u128 = 100000000000000000000000u128;
-pub const DEFAULT_POOL_AMOUNT_D: u128 = 6000000000000000000000u128;
+pub const DEFAULT_POOL_AMOUNT_A: u32 = 100000;
+pub const DEFAULT_POOL_AMOUNT_B: u32 = 6000;
+pub const DEFAULT_POOL_AMOUNT_C: u32 = 100000;
+pub const DEFAULT_POOL_AMOUNT_D: u32 = 6000;
 
 /// The order amount for orders selling token "A" for "B".
-pub const AB_ORDER_AMOUNT: u128 = 50000000000000000000u128;
+pub const AB_ORDER_AMOUNT: u32 = 50;
 
 /// The order amount for orders selling token "C" for "D".
-pub const CD_ORDER_AMOUNT: u128 = 40000000000000000000u128;
+pub const CD_ORDER_AMOUNT: u32 = 40;
 
-pub const ETH_ORDER_AMOUNT: u128 = 40000000000000000000u128;
+pub const ETH_ORDER_AMOUNT: u32 = 40;
 
 /// With the default amounts defined above, this is the expected score range for
 /// both buy and sell orders.
-pub const DEFAULT_SCORE_MIN: u128 = 2000000000000000000u128;
-pub const DEFAULT_SCORE_MAX: u128 = 500000000000000000000000000000u128;
+pub const DEFAULT_SCORE_MIN: u32 = 2;
+pub const DEFAULT_SCORE_MAX: u64 = 500000000000;
 
 /// The default solver fee for limit orders.
-pub const DEFAULT_SOLVER_FEE: u128 = 100u128;
+pub const DEFAULT_SOLVER_FEE: f64 = 1e-16;
 
 /// The default maximum value to be payout out to solver per solution
-pub const DEFAULT_SCORE_CAP: u128 = 10000000000000000u128;
+pub const DEFAULT_SCORE_CAP: f64 = 0.01;
+
+pub trait ToWei {
```

Comment:
nit: from the signature this looks more like an `into_wei`. Also, is this change even needed now (i don't see where we are using the float representation)

Reply:
I was experimenting with test values and made mistakes a couple of times with extra or missing zeros. Really liked the approach from e2e tests and tried to use it here. It can be reverted if it looks redundant at the moment. The float representation is used in constants only.

Reply:
ok, could be its own PR then 😄

Reply:
Reverted the change